queries/tokenToTokenPriceQuery.ts

```diff
@@ -80,8 +80,10 @@ export async function tokenToTokenPriceQueryWithPools({
       pricingQueries.push(
         getTokenForTokenPrice({
           tokenAmount: convertedTokenAmount,
-          swapAddress: passThroughPool.inputPool.swap_address,
-          outputSwapAddress: passThroughPool.outputPool.swap_address,
+          tokenA,
+          tokenB,
+          swap: passThroughPool.inputPool,
+          outputSwap: passThroughPool.outputPool,
```

Comment:
Think it's best to keep it `swap`/`outputSwap` or would rather make it more consistent like you do it in `useTokenSwap`?